acceptance_tests/features/forgotten_password_respondent.feature

```diff
@@ -0,0 +1,15 @@
+Feature: External user signs in
```

Comment:
I'd probably make this feature more explicit that these scenarios are specifically about recovering passwords